src/app/lib/analyticsUtils/index.js

```diff
@@ -295,4 +301,36 @@ export const getThingAttributes = (attribute, articleData) => {
   return null;
 };
 
+export const getCampaignType = () => {
+  if (!onClient()) return null;
+
+  // Gets the query string parameters from the current url parsing them as an object
+  const { query, hash } = new Url(window.location.href, true);
```

Comment:
Just a quick question for my understanding about the `true` argument being passed to new Url. 

From the [docs](https://www.npmjs.com/package/url-parse) it says:

> The constructor takes the following arguments:

> - url (String): A string representing an absolute or relative URL.
> - baseURL (Object | String): An object or string representing the base URL to use in case url is a relative URL. This argument is optional and defaults to location in the browser.
> - parser (Boolean | Function): This argument is optional and specifies how to parse the query string. By default it is false so the query string is not parsed. If you pass true the query string is parsed using the embedded querystringify module. If you pass a function the query string will be parsed using this function.

`true` is the second argument being passed to the Url constructor but it doesn't seem to be the baseUrl argument, but rather the parser argument..?

Reply:
So the second param of `baseUrl` is optional, however it also has a default value of `window.location` when in the browser.

In this case passing in `true` as the second param does not override the baseUrl as it expects either a string or an object not a boolean value.

So in this case passing in `true` is us asking for it to parse the URL and give us a object of the parsed values.

Also taken from another example in their docs.

```
var parse = require('url-parse')
  , url = parse('https://github.com/foo/bar', true);
```

Very good question though 👍

Reply:
ahh I see - thanks a lot for explaining!

Reply:
Will this be constructed for amp as well ? Based on [how we construct the href](https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.js#L107-L117) value and [how it's used](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/atiUrl/index.js#L45), I reckon `window.location.href` won't be available on amp. so this should vary based on platform, I think 🤔